d2/src/main/java/com/linkedin/d2/xds/XdsClient.java

```diff
@@ -105,18 +105,18 @@ public String getVersion()
     }
   }
 
-  static final class D2NodeMapUpdate implements ResourceUpdate
+  static final class D2URIMapUpdate implements ResourceUpdate
   {
     String _version;
-    Map<String, XdsD2.D2Node> _nodeDataMap;
+    Map<String, XdsD2.D2URI> _nodeDataMap;
```

Comment:
nit pick, should we also update the variable name?  _uriDataMap?

Reply:
Sure makes sense